test/integration/patients/worklist/schedule.js

```diff
@@ -242,7 +245,7 @@ context('schedule page', function() {
 
     cy
       .url()
-      .should('contain', 'patient/1/action/1')
+      .should('contain', 'flow/1/action/1')
```

Comment:
It looks like this change reduced coverage by no longer testing if a patient action links to the correct place.